hypertrace-graphql-entity-schema/src/main/java/org/hypertrace/graphql/entity/dao/EntityDaoModule.java

```diff
@@ -43,8 +43,8 @@ protected void configure() {
     bind(BaselineDao.class).to(GatewayBaselineDao.class);
     requireBinding(CallCredentials.class);
     requireBinding(GraphQlServiceConfig.class);
-    requireBinding(GraphQlRequestContextBuilder.class);
```

Comment:
Is this intentional?

Reply:
Yes. Since I was in there, I switched from the deprecated `GraphQlGrpcContextBuilder` to `GrpcContextBuilder`. When I went to switch the binding declaration (which is really just for documentation purposes), I noticed we had accidentally required `GraphQlRequestContextBuilder` (converts from the raw http to our internal format, which this module doesn't do), instead of `GraphQlGrpcContextBuilder` (converts from our internal format to a grpc format - which this module does do, and was swapped out)